{"uid":"8a360edf4af9e3df","name":"test_list_dt_06_10","fullName":"tests.test_checklist.TestPromotionChecklist#test_list_dt_06_10","historyId":"73b0ce9bb1efc7bf3702d12b8902e199","time":{"start":1739721715626,"stop":1739721875283,"duration":159657},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist","time":{"start":1739712259353,"stop":1739712259353,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"passed","steps":[{"name":"Click authorisation button","time":{"start":1739721716548,"stop":1739721722456,"duration":5908},"status":"passed","steps":[{"name":"Get authorisation button","time":{"start":1739721716548,"stop":1739721716559,"duration":11},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Source select","time":{"start":1739721722990,"stop":1739721723108,"duration":118},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"source","value":"'10'"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721724581,"stop":1739721724673,"duration":92},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"1"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721724673,"stop":1739721732473,"duration":7800},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"\nMetadata = PROPERTY Ok âœ…\n","time":{"start":1739721732493,"stop":1739721732493,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721732516,"stop":1739721732640,"duration":124},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"2"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721732640,"stop":1739721742665,"duration":10025},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class CUSTOMPROPERTY is not mapped","time":{"start":1739721742666,"stop":1739721742666,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721742688,"stop":1739721742816,"duration":128},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"3"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721742816,"stop":1739721752865,"duration":10049},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class DATASYSTEM is not mapped","time":{"start":1739721752866,"stop":1739721752866,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721752889,"stop":1739721753007,"duration":118},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"4"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721753007,"stop":1739721763020,"duration":10013},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class ENUMERATION is not mapped","time":{"start":1739721763021,"stop":1739721763021,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721763043,"stop":1739721763162,"duration":119},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721763162,"stop":1739721773184,"duration":10022},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYROOMS is not mapped","time":{"start":1739721773185,"stop":1739721773185,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721773207,"stop":1739721773328,"duration":121},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"6"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721773328,"stop":1739721783350,"duration":10022},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYUNITTYPES is not mapped","time":{"start":1739721783351,"stop":1739721783351,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721783372,"stop":1739721783496,"duration":124},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"7"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721783496,"stop":1739721793549,"duration":10053},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYBUILDING is not mapped","time":{"start":1739721793550,"stop":1739721793550,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721793571,"stop":1739721793711,"duration":140},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"8"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721793711,"stop":1739721803770,"duration":10059},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLASTMARKETSALE is not mapped","time":{"start":1739721803771,"stop":1739721803771,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721803806,"stop":1739721803937,"duration":131},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"9"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721803937,"stop":1739721813968,"duration":10031},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLOCATION is not mapped","time":{"start":1739721813969,"stop":1739721813969,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721814001,"stop":1739721814134,"duration":133},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"10"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721814134,"stop":1739721824184,"duration":10050},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIP is not mapped","time":{"start":1739721824185,"stop":1739721824185,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721824219,"stop":1739721824353,"duration":134},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"11"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721824354,"stop":1739721834387,"duration":10033},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIPTRANSFER is not mapped","time":{"start":1739721834388,"stop":1739721834388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721834423,"stop":1739721834561,"duration":138},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"12"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721834561,"stop":1739721844610,"duration":10049},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYPRIORSALE is not mapped","time":{"start":1739721844611,"stop":1739721844611,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721844645,"stop":1739721844789,"duration":144},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"13"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721844789,"stop":1739721854810,"duration":10021},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSEARCH is not mapped","time":{"start":1739721854811,"stop":1739721854811,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721854842,"stop":1739721854991,"duration":149},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"14"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721854991,"stop":1739721865007,"duration":10016},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSITE is not mapped","time":{"start":1739721865008,"stop":1739721865008,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739721865041,"stop":1739721865189,"duration":148},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"15"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739721865189,"stop":1739721875219,"duration":10030},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x56212b0f209a <unknown>\n#1 0x56212abb98b0 <unknown>\n#2 0x56212ac0ab99 <unknown>\n#3 0x56212ac0ad21 <unknown>\n#4 0x56212ac59854 <unknown>\n#5 0x56212ac309ad <unknown>\n#6 0x56212ac56b8b <unknown>\n#7 0x56212ac30753 <unknown>\n#8 0x56212abfc38e <unknown>\n#9 0x56212abfdb51 <unknown>\n#10 0x56212b0bb76b <unknown>\n#11 0x56212b0bf6f2 <unknown>\n#12 0x56212b0a78fc <unknown>\n#13 0x56212b0c02e4 <unknown>\n#14 0x56212b08b9cf <unknown>\n#15 0x56212b0e0cd8 <unknown>\n#16 0x56212b0e0eb6 <unknown>\n#17 0x56212b0f0f16 <unknown>\n#18 0x7f6977a9caa4 <unknown>\n#19 0x7f6977b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYTAXASSESSMENT is not mapped","time":{"start":1739721875220,"stop":1739721875220,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":48,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist::0","time":{"start":1739727916514,"stop":1739727916514,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.filterwarnings('ignore::DeprecationWarning')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_checklist"},{"name":"subSuite","value":"TestPromotionChecklist"},{"name":"host","value":"fv-az1392-979"},{"name":"thread","value":"3356-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_checklist"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@pytest.mark.filterwarnings('ignore::DeprecationWarning')"]},"source":"8a360edf4af9e3df.json","parameterValues":[]}